### ISSUE TITLE
Disable AsyncHttpResponseHandler Logs

### ISSUE THREAD

**Issue by davityle** [loopj/android-async-http#757]
Hey, It would be great if we could disable this line of code
```
public void onProgress(int bytesWritten, int totalSize) {
        Log.v(LOG_TAG, String.format("Progress %d from %d (%2.0f%%)", bytesWritten, totalSize, (totalSize > 0) ? (bytesWritten * 1.0 / totalSize) * 100 : -1));
    }
```

in the AsyncHttpResponseHandler. My logs always look like this.
```
12-10 10:56:12.021   V/AsyncHttpResponseHandler﹕ Progress 7900 from 299844 ( 3%)
12-10 10:56:12.021   V/AsyncHttpResponseHandler﹕ Progress 11996 from 299844 ( 4%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 16092 from 299844 ( 5%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 20188 from 299844 ( 7%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 24284 from 299844 ( 8%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 28380 from 299844 ( 9%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 32476 from 299844 (11%)
12-10 10:56:12.031   V/AsyncHttpResponseHandler﹕ Progress 34752 from 299844 (12%)
12-10 10:56:12.041   V/AsyncHttpResponseHandler﹕ Progress 36200 from 299844 (12%)
12-10 10:56:12.051   V/AsyncHttpResponseHandler﹕ Progress 40296 from 299844 (13%)
12-10 10:56:12.051   V/AsyncHttpResponseHandler﹕ Progress 44392 from 299844 (15%)
12-10 10:56:12.051   V/AsyncHttpResponseHandler﹕ Progress 48488 from 299844 (16%)
12-10 10:56:12.051   V/AsyncHttpResponseHandler﹕ Progress 52584 from 299844 (18%)
12-10 10:56:12.061   V/AsyncHttpResponseHandler﹕ Progress 56680 from 299844 (19%)
12-10 10:56:12.061   V/AsyncHttpResponseHandler﹕ Progress 60776 from 299844 (20%)
12-10 10:56:12.061   V/AsyncHttpResponseHandler﹕ Progress 64872 from 299844 (22%)
12-10 10:56:12.071   V/AsyncHttpResponseHandler﹕ Progress 68968 from 299844 (23%)
12-10 10:56:12.071   V/AsyncHttpResponseHandler﹕ Progress 4096 from 299844 ( 1%)
12-10 10:56:12.081   V/AsyncHttpResponseHandler﹕ Progress 7900 from 299844 ( 3%)
12-10 10:56:12.091   V/AsyncHttpResponseHandler﹕ Progress 11996 from 299844 ( 4%)
12-10 10:56:12.091   V/AsyncHttpResponseHandler﹕ Progress 16092 from 299844 ( 5%)
12-10 10:56:12.101   V/AsyncHttpResponseHandler﹕ Progress 20188 from 299844 ( 7%)
12-10 10:56:12.101   V/AsyncHttpResponseHandler﹕ Progress 24284 from 299844 ( 8%)
12-10 10:56:12.101   V/AsyncHttpResponseHandler﹕ Progress 28380 from 299844 ( 9%)
12-10 10:56:12.111   V/AsyncHttpResponseHandler﹕ Progress 32476 from 299844 (11%)
12-10 10:56:12.111   V/AsyncHttpResponseHandler﹕ Progress 36572 from 299844 (12%)
12-10 10:56:12.111   V/AsyncHttpResponseHandler﹕ Progress 40668 from 299844 (14%)
12-10 10:56:12.111   V/AsyncHttpResponseHandler﹕ Progress 73064 from 299844 (24%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 44764 from 299844 (15%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 48860 from 299844 (16%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 52956 from 299844 (18%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 57052 from 299844 (19%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 61148 from 299844 (20%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 65244 from 299844 (22%)
12-10 10:56:12.121   V/AsyncHttpResponseHandler﹕ Progress 69340 from 299844 (23%)
12-10 10:56:12.131   V/AsyncHttpResponseHandler﹕ Progress 73436 from 299844 (24%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 77532 from 299844 (26%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 81628 from 299844 (27%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 85724 from 299844 (29%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 89820 from 299844 (30%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 93916 from 299844 (31%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 98012 from 299844 (33%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 77160 from 299844 (26%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 81256 from 299844 (27%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 85352 from 299844 (28%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 89448 from 299844 (30%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 92672 from 299844 (31%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 96768 from 299844 (32%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 100864 from 299844 (34%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 104960 from 299844 (35%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 109056 from 299844 (36%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 113152 from 299844 (38%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 114392 from 299844 (38%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 118488 from 299844 (40%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 122584 from 299844 (41%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 126680 from 299844 (42%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 130776 from 299844 (44%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 134872 from 299844 (45%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 138968 from 299844 (46%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 140456 from 299844 (47%)
12-10 10:56:12.141   V/AsyncHttpResponseHandler﹕ Progress 102108 from 299844 (34%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 106204 from 299844 (35%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 110300 from 299844 (37%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 141904 from 299844 (47%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 114396 from 299844 (38%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 118492 from 299844 (40%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 122588 from 299844 (41%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 126684 from 299844 (42%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 130780 from 299844 (44%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 143352 from 299844 (48%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 147448 from 299844 (49%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 151544 from 299844 (51%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 155640 from 299844 (52%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 159736 from 299844 (53%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 163832 from 299844 (55%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 167928 from 299844 (56%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 172024 from 299844 (57%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 176120 from 299844 (59%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 180216 from 299844 (60%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 183896 from 299844 (61%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 134876 from 299844 (45%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 138972 from 299844 (46%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 143068 from 299844 (48%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 147164 from 299844 (49%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 151260 from 299844 (50%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 155356 from 299844 (52%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 187992 from 299844 (63%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 192088 from 299844 (64%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 196184 from 299844 (65%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 200280 from 299844 (67%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 204376 from 299844 (68%)
12-10 10:56:12.151   V/AsyncHttpResponseHandler﹕ Progress 208472 from 299844 (70%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 211408 from 299844 (71%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 159452 from 299844 (53%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 163548 from 299844 (55%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 167644 from 299844 (56%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 171740 from 299844 (57%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 175836 from 299844 (59%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 179932 from 299844 (60%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 184028 from 299844 (61%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 188124 from 299844 (63%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 192220 from 299844 (64%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 196316 from 299844 (65%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 200412 from 299844 (67%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 204508 from 299844 (68%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 208604 from 299844 (70%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 215504 from 299844 (72%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 219600 from 299844 (73%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 223696 from 299844 (75%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 227792 from 299844 (76%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 231680 from 299844 (77%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 212700 from 299844 (71%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 216796 from 299844 (72%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 220892 from 299844 (74%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 224988 from 299844 (75%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 229084 from 299844 (76%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 235776 from 299844 (79%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 233180 from 299844 (78%)
12-10 10:56:12.161   V/AsyncHttpResponseHandler﹕ Progress 237276 from 299844 (79%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 241372 from 299844 (80%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 245468 from 299844 (82%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 249564 from 299844 (83%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 253660 from 299844 (85%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 257756 from 299844 (86%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 261852 from 299844 (87%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 265948 from 299844 (89%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 270044 from 299844 (90%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 274140 from 299844 (91%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 278236 from 299844 (93%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 282332 from 299844 (94%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 286428 from 299844 (96%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 290524 from 299844 (97%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 294620 from 299844 (98%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 298716 from 299844 (100%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 299844 from 299844 (100%)
12-10 10:56:12.171   V/AsyncHttpResponseHandler﹕ Progress 239872 from 299844 (80%)
12-10 10:56:12.261   D/dalvikvm﹕ GC_FOR_ALLOC freed 3542K, 37% free 23251K/36372K, paused 25ms, total 25ms
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 243968 from 299844 (81%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 248064 from 299844 (83%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 252160 from 299844 (84%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 256256 from 299844 (85%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 260352 from 299844 (87%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 264448 from 299844 (88%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 268544 from 299844 (90%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 272640 from 299844 (91%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 276736 from 299844 (92%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 280832 from 299844 (94%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 284928 from 299844 (95%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 289024 from 299844 (96%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 293120 from 299844 (98%)
12-10 10:56:12.301   V/AsyncHttpResponseHandler﹕ Progress 297216 from 299844 (99%)
```

**Comment by smarek** [comment]
Well, feature was just now provided and will be released with 1.4.8 version in few days.
Use ie. like this:

```java
// To disable all default logging messages
AsyncHttpClient ahc = new AsyncHttpClient();
ahc.setLoggingEnabled(false);
```
or
```java
// To enable logging just error level
AsyncHttpClient ahc = new AsyncHttpClient();
ahc.setLoggingLevel(Log.ERROR);
```

or you can provide your own implementation (ie. for integration with Timber, Crashlytics, ...) via:
```java
AsyncHttpClient ahc = new AsyncHttpClient();
ahc.setLogInterface(myLogInterfaceImplementation);
```

**Comment by smarek** [comment]
Respective commits: 9164374f8e25f7d1103f2ac9e66f27ea8e5c0e95 and 908e251a3a21732eb83a4e460bfdc3e4268fe701
